src/features/drawing/piano.ts

```diff
@@ -1,9 +1,8 @@
+import { roundCorner, roundRect } from '@/features/drawing'
 import { getKey, getOctave, isBlack, isWhite } from '@/features/theory'
-import { roundRect, roundCorner } from '@/features/drawing'
 import midiState from '../midi'
 import { isPointerDown } from '../pointer'
-import { isNumber } from '@/utils'
```

Comment:
I think removing `isNumber` is very breaking, but it works if you add this line back

Reply:
This change was made by prettier! 🤷‍♂️